website/content/docs/using-netlify-cms-with-gatsby.md

```diff
@@ -0,0 +1,119 @@
+# Using Netlify CMS with Gatsby
+
+This guide will help you get started using Netlify CMS and Gatsby.
+
+To get up and running with Gatsby, you’ll need to have [Node.js](https://nodejs.org/) installed on your computer. _Note: Gatsby's minimum supported Node.js version is Node 8._
+
+## Create a new Gatsby site
+
+Let's create a new site using the default Gatsby Starter Blog. Run the following commands in the terminal, in the folder where you'd like to create the blog:
+
+```sh
+npm install -g gatsby-cli
+gatsby new blog https://github.com/gatsbyjs/gatsby-starter-blog
+cd blog
+```
+
+## Get to know Gatsby
+
+In your favorite code editor, open up the code generated for your "Gatsby Starter Blog" site, and take a look at the `content` directory.
+
+You will see that there are multiple Markdown files that represent blog posts. Open one `.md` file and you will see something like this:
+
+```yml
+---
+title: New Beginnings
+date: "2015-05-28T22:40:32.169Z"
+description: This is an optional description for SEO and Open Graph purposes, rather than the default generated excerpt.
+---
+
+Far far away, behind the word mountains, far from the countries Vokalia and
+Consonantia, there live the blind texts.
+```
+
+We can see above that each blog post has a title, a date, a description and a body. Now, let's recreate this using Netlify CMS. 
+
+## Add Netlify CMS to your site
+
+First let's install some dependencies. We'll need `netlify-cms` and `gatsby-plugin-netlify-cms`. Run the following command in the terminal at the root of your site:
+
+```sh
+npm install --save netlify-cms gatsby-plugin-netlify-cms
+```
+
+### Configuration
+
+For the purpose of this guide we will deploy to Netlify from a GitHub repository which requires the minimum configuration.
+
+Create a `config.yml` file in the directory structure you see below:
+
+```sh
+├── static
+│   ├── admin
+│   │   ├── config.yml
+```
+
+In your `config.yml` file paste the following configuration:
+
+```yml
+backend:
+  name: git-gateway
+  branch: master
+
+media_folder: static/img
+public_folder: /img
+
+collections:
+  - name: "blog"
+    label: "Blog"
+    folder: "content/blog"
+    create: true
+    slug: "{{year}}-{{month}}-{{day}}-{{slug}}"
+    editor:
+     preview: false
+    fields:
+      - { label: "Title", name: "title", widget: "string" }
+      - { label: "Publish Date", name: "date", widget: "datetime" }
+      - { label: "Description", name: "description", widget: "string" }
+      - { label: "Body", name: "body", widget: "markdown" }
+```
+
+Finally, add the plugin to your `gatsby-config.js`.
+
+```javascript
+plugins: [`gatsby-plugin-netlify-cms`]
+```
+
+### Push to GitHub
+
+It's now time to commit your changes and push to GitHub. The Gatsby starter initializes Git automatically for you, so you only need to do:
+
+```bash
+git add .
+git commit -m "Initial Commit"
+git remote add origin git@github.com:user/repo.git
```

Comment:
I'd switch this to an https url, so folks without SSH keys won't get tripped up. For those who use SSH, we can be pretty safe assuming they'll know to translate the url.

I'd also recommend giving a bit more guidance in the url, like: 
`https://github.com/YOUR_USERNAME/NEW_REPO_NAME.git`

Reply:
fixed 👌